packages/wonder-blocks-form/src/components/text-field.md

```diff
@@ -9,15 +9,15 @@ class TextFieldExample extends React.Component {
         this.state = {
             value: "",
         };
-        this.handleOnChange = this.handleOnChange.bind(this);
-        this.handleOnKeyDown = this.handleOnKeyDown.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.handleKeyDown = this.handleKeyDown.bind(this);
     }
 
-    handleOnChange(newValue) {
+    handleChange(newValue) {
```

Comment:
style nit: you should be able to replace
```
this.handleChange = this.handleChange.bind(this);
```
with
```
handleChange = (newValue) => { ... }
```
Are class properties not working in the examples?

Reply:
Oh yes, sadly that's an issue with Styleguidist. Good news is that we no longer have to support that in Storybook :).